src/condor_utils/test_classad_funcs.cpp

```diff
@@ -0,0 +1,536 @@
+		"EXPECT: false     FROM stringListMember(\"\",sl3cb)\n"
+		"EXPECT: false     FROM stringListMember(un,sl3cb)\n"
+		"EXPECT: undefined FROM stringListMember(un,un)\n"
+		"EXPECT: false     FROM stringListMember(un,empty)\n"
+		"EXPECT: false     FROM stringListMember(empty,un)\n"
+		"EXPECT: false     FROM stringListMember(empty,empty)\n"
+
+		"EXPECT: false     FROM stringListMember(b,join(com,l3))\n"
+		"EXPECT: true      FROM stringListMember(b,toupper(join(com,l3)))\n"
+		"EXPECT: true      FROM stringListIMember(b,join(com,l3))\n"
+	};
+
+	StringLiteralInputStream stm(tests);
+	testing_from("stringListMember", stm, verbose);
+}
+
+void testing_stringListSubsetMatch(int verbose)
+{
+	const char * tests = {
+		"MY:\n"
+		"  l3 = {\"aa\",\"bb\",\"cc\"}\n"
+		"  sl3 = \"aa bb cc\"\n"
+		"  sl3c = \"cc,bb,aa\"\n"
+		"  sl3ce = \"cc,bb,aa,,\"\n"
+		"  sl3cb = \"cc,bb,aa,,\"\n"
+		"  sl3ws = \"cc, BB, , aa\"\n"
+		"  sl2b = \"bb,aa\"\n"
+		"  A = \"aa\"\n"
+		"  B = \"BB\"\n"
+		"  com = \",\"\n"
+		"  sp = \" \"\n"
+		"  empty = \"\"\n"
+
+		"EXPECT: true      FROM stringListSubsetMatch(a,sl3)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(a,sl3c)\n"
+		"EXPECT: false     FROM stringListSubsetMatch(B,sl3)\n"
+		"EXPECT: false     FROM stringListSubsetMatch(B,sl3c)\n"
+		"EXPECT: true      FROM stringListISubsetMatch(B,sl3)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(s12b,sl3)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(s12b,sl3c)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(B,sl3ws)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(B,sl3ws,com)\n"
+		"EXPECT: false     FROM stringListSubsetMatch(sp,sl3ws,com)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(empty,sl3ws,com)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(UN,sl3ws,com)\n"
+		"EXPECT: true      FROM stringListISubsetMatch(s12b,sl3ce)\n"
+
+		"EXPECT: error     FROM stringListSubsetMatch()\n"
+		"EXPECT: error     FROM stringListSubsetMatch(a,l3)\n"
+		"EXPECT: error     FROM stringListSubsetMatch(a,b,A,B)\n"
+
+		"EXPECT: true      FROM stringListSubsetMatch(\"\",sl3c)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(\"\",sl3ce)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(\"\",sl3cb)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(un,sl3cb)\n"
+		"EXPECT: false     FROM stringListSubsetMatch(B,UN)\n"
+		"EXPECT: undefined FROM stringListSubsetMatch(un,un)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(un,empty)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(empty,un)\n"
+		"EXPECT: true      FROM stringListSubsetMatch(empty,empty)\n"
+
+		"EXPECT: false     FROM stringListSubsetMatch(b,join(com,l3))\n"
+		"EXPECT: true      FROM stringListSubsetMatch(b,toupper(join(com,l3)))\n"
+		"EXPECT: true      FROM stringListISubsetMatch(b,join(com,l3))\n"
+	};
+
+	StringLiteralInputStream stm(tests);
+	testing_from("stringListSubsetMatch", stm, verbose);
+}
+
+int main(int argc, char **argv)
```

Comment:
how about
`int main(int /*argc*/, char **argv)`

to remove used argc warning: